tags/interceptors.go

```diff
@@ -65,7 +82,7 @@ func (w *wrappedStream) RecvMsg(m interface{}) error {
 	return err
 }
 
-func newTagsForCtx(ctx context.Context) context.Context {
+func NewTagsForCtx(ctx context.Context) context.Context {
```

Comment:
can we drop this back to being internal to the package?

Reply:
This method is now also used in the tracing interceptor https://github.com/grpc-ecosystem/go-grpc-middleware/pull/123#discussion-diff-161192910R118